src/invoices/InvoiceLineDetails/InvoiceLineDetails.js

```diff
@@ -160,6 +162,14 @@ const InvoiceLineDetails = ({
                 currency={currency}
               />
             </Accordion>
+            {Boolean(otherRelatedInvoiceLines.length) && (
+              <Accordion
+                id={SECTIONS_INVOICE_LINE.otherRelatedInvoiceLines}
+                label={<FormattedMessage id="ui-invoice.otherRelatedInvoiceLines" />}
+              >
+                <OtherRelatedInvoiceLines invoiceLines={otherRelatedInvoiceLines} />
```

Comment:
can otherRelatedInvoiceLines be fetched in OtherRelatedInvoiceLines?

Reply:
Yes, made changes